app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx

```diff
@@ -101,7 +111,19 @@ export class DashboardPageHeader extends Component {
     );
 
   render() {
-    const { activeItemId, eventsInfo } = this.props;
+    const {
+      activeItemId,
+      eventsInfo,
+      dashboardsToDisplay,
+      isLoading,
+      totalDashboards,
+    } = this.props;
+
+    const isCalculateExtraItems = totalDashboards > DASHBOARDS_LIMIT;
+    const extraItemsCount = isCalculateExtraItems ? totalDashboards - DASHBOARDS_LIMIT : 0;
+    const isAboveLimit = dashboardsToDisplay.length + extraItemsCount >= DASHBOARDS_LIMIT;
```

Comment:
Why not to just check that?
```javascript
const isAboveLimit = totalDashboards >= DASHBOARDS_LIMIT;
```

Reply:
`const isAboveLimit = dashboardsToDisplay.length + extraItemsCount >= DASHBOARDS_LIMIT;`
That line covered the next scenario.
A user has 310 dashboards, the button `add dashboard` is disabled.  The user starts to delete his dashboards, count of totalDashboards didn't change, because we don't send request after we delete the dashboard.
But the count of dashBoardsToDisplay is decreasing.